jest-common/src/main/java/io/searchbox/core/SearchResult.java

```diff
@@ -176,6 +177,21 @@ protected JsonElement getPath(String[] path) {
         return facets;
     }
 
+    public Aggregation getAggregations() {
+        if (jsonObject != null) {
+            JsonObject aggregationsMap;
+            if (jsonObject.has("aggregations")) {
+                aggregationsMap = (JsonObject) jsonObject.get("aggregations");
+            } else if (jsonObject.has("aggs")) {
+                aggregationsMap = (JsonObject) jsonObject.get("aggs");
+            } else {
+                return new Aggregation("aggs", new JsonObject());
+            }
+            return new Aggregation("aggs", aggregationsMap);
+        }
+        return new Aggregation("aggs", new JsonObject());
+    }
```

Comment:
Since there are already multiple exit points, refactoring as below would make it much more easy to read:

````
if (jsonObject == null) return new Aggregation("aggs", new JsonObject());
if (jsonObject.has("aggregations")) return new Aggregation("aggs", jsonObject.getAsJsonObject("aggregations"));
if (jsonObject.has("aggs")) return new Aggregation("aggs", jsonObject.getAsJsonObject("aggs"));

return new Aggregation("aggs", new JsonObject());
```